src/EA.Weee.Web/Areas/Scheme/Attributes/ValidatePcsObligationsEnabledAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace EA.Weee.Web.Areas.Scheme.Attributes
+{
+    using System;
+    using System.Web.Mvc;
+    using Api.Client;
+    using Services;
+
+    public class ValidatePcsObligationsEnabledAttribute : ActionFilterAttribute
```

Comment:
Can we add unit tests for ValidatePcsObligationsEnabledAttribute similar to ValidateEvidenceEnabledAttributeTests